compliance/repository/src/test/java/org/eclipse/rdf4j/repository/sparql/SPARQLRepositorySparqlUpdateTest.java

```diff
@@ -52,3 +64,42 @@
 //		System.err.println("temporarily disabled testAutoCommitHandling() for HTTPRepository");
 //	}
 //}
+public class SPARQLRepositorySparqlUpdateTest extends TestCase {
```

Comment:
Ah, I only just now noticed that this was an existing, commented-out test class that you added your test to. I don't think that's quite right, let me have a closer look and I'll try and give some advice on how to better organize this.

Reply:
Yeah, I think you should undo this, leave this old class commented out, and move your tests to the test suite. See my other comment, below, for details.